tests/legacy/ui/test_no_selfclose_create_import_publish_module_and_collection.py

```diff
@@ -0,0 +1,119 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import NoSuchElementException
+
+from tests import markers
+
+from pages.legacy.login_form import LoginForm
+from pages.legacy.module_edit import ModuleEdit
+from pages.legacy.collection_edit import CollectionEdit
+from pages.legacy.content_published import ContentPublished
+from pages.archive.legacy_content import LegacyContent
+
+from regions.legacy.collection import Collection
+from regions.legacy.module import Module
+
+
+class TestNoSelfCloseCreateImportPublishModuleAndCollection(object):
+    _module_temp_id = None
+    _module_id = None
+    _collection_temp_id = None
+
+    @markers.legacy
+    @markers.slow
+    def test_noselfclose_create_module(self, legacy_base_url, legacy_username, legacy_password, selenium):
+        # GIVEN a logged in user on their dashboard
+        login_page = LoginForm(selenium, legacy_base_url).open()
+        my_cnx = login_page.login(legacy_username, legacy_password)
+
+        # WHEN the user clicks to create a new module, agrees to the license and fills in the Title
+        cc_license = my_cnx.create_module()
+        metadata_edit = cc_license.agree().submit()
+        module_edit = metadata_edit.fill_in_title("CNX Automation Test Module").submit()
+
+        # THEN the user is brought to the module editor
+        assert type(module_edit) is ModuleEdit
+        assert module_edit.title == "CNX Automation Test Module"
```

Comment:
Might want to title it in regards to self closing tags so we can differentiate from other tests.

Reply:
changed on https://github.com/openstax/cnx-automation/pull/589/commits/2885e2f2c2c6fca3fcbe25d72696853524516e46